app/src/main/java/com/example/caregiver/AddTask.java

```diff
@@ -92,7 +84,7 @@ protected void onCreate(Bundle savedInstanceState) {
         createSpinners();
 
         // navigate back to dashboard
-        Button backButton = findViewById(R.id.taskBackButton);
+        Button backButton = findViewById(R.id.taskCancelButton);
```

Comment:
This is because I refactored it and had to change it back but forgot what it was called lol 
I tested AddTask functionality to make sure that's still working well!